action.yml

```diff
@@ -42,6 +46,9 @@ inputs:
     description: Maximum acceptable proportion of pixels that exceed the `max-allowed-color-difference`, as a number between 0 and 1. Differences in a smaller or equal proportional of pixels to this will be ignored. Defaults to 0.00001, or about 20 pixels in a 1920 x 1080 px screenshot.
     required: true
     default: 0.00001
+  test-suite-id:
+    description: Used to differentiate between different Meticulous test suites if you run this action multiple times as part of a single workflow.
+    required: false
```

Comment:
Added this mainly just for our tests:

<img width="966" alt="image" src="https://user-images.githubusercontent.com/793481/226700893-2872a8e3-15d4-4eac-823a-ef9a561794f5.png">